src/trades/dto/create-trade.input.ts

```diff
@@ -0,0 +1,25 @@
+import { Field, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class CreateTradeInput {
+  @Field()
+  trade_uuid: string;
+
+  @Field()
+  sender_uuid: string;
+
+  @Field()
+  receiver_uuid: string;
+
+  @Field()
+  state: string;
+
+  @Field(() => [String])
+  sender_items: string[];
+
+  @Field(() => [String])
+  receiver_items: string[];
+
+  @Field(() => [String])
+  trade_history: string[];
```

Comment:
I'm thinking we just store the id of the previous trade instead of the whole history, because then it's too hard to maintain